acquia_lift_profiles/acquia_lift_profiles.admin.inc

```diff
@@ -101,43 +101,37 @@ function acquia_lift_profiles_admin_form($form, &$form_state) {
     )),
     '#default_value' => variable_get('acquia_lift_profiles_default_identity_type', ''),
   );
-  $vocabularies = taxonomy_get_vocabularies();
-  $mappings = variable_get('acquia_lift_profiles_vocabulary_mappings', array());
-  $vocab_options = array('' => t('Select...'));
-  foreach ($vocabularies as $vid => $voc) {
-    $vocab_options[$vid] = $voc->name;
-  }
+
+  // Retrieving all context options.
+  module_load_include('inc', 'personalize', 'personalize.admin');
+  $groups = personalize_get_grouped_context_options(NULL, TRUE, array('acquia_lift_profiles_context', 'acquia_lift_context'));
+
+  // Vocabulary mappings.
   $form['acquia_lift_profiles_vocabulary_mappings'] = array(
     '#title' => t('Vocabulary Mappings'),
+    '#description' => t('For each vocabularies available in Acquia Lift Profiles, you can map a visitor context, whose value will then be sent as the value for that vocabulary.'),
```

Comment:
They are not vocabularies in Acquia Lift Profiles. Let's say something like "For the 'site section', 'content keywords' and 'persona' fields in Acquia Lift Profiles, you can map a visitor context..."